ipld/get_namespaced_shares.go

```diff
@@ -77,11 +77,11 @@ type fetchedBounds struct {
 	highest int64
 }
 
-// update checks if the passed index is outside the current bounds,
+// update checks if the passed pos is outside the current bounds,
```

Comment:
the renames here are incorrect, because they are referring to the argument of update